resources/views/architects/index.blade.php

```diff
@@ -0,0 +1,18 @@
+@extends('layouts.app')
+@section('title',  __('architects.title'))
+
+@section('content')
+
+@include('components.header.header')
+
+<div class="container mx-auto px-6">
+    <x-intro
+        :title="trans('architects.title')"
+        text="“{{ __('app.intro') }}”"
+    />
+    <architects-overview />
```

Comment:
[boulder] This would be a good place to pass translations for the filters
```suggestion
    <architects-overview 
      :authorizationLabels="{
        AA: {{ trans('architects.authorizations.AA') }},
        ...
      }"
    />
```